src/openrct2-ui/windows/DemolishRidePrompt.cpp

```diff
@@ -88,41 +88,39 @@ static rct_window_event_list window_ride_demolish_events = {
 };
 // clang-format on
 
-static rct_window_event_list window_ride_refurbish_events = {
-    nullptr,
-    window_ride_refurbish_mouseup,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    window_ride_refurbish_paint,
-    nullptr
-};
+static rct_window_event_list window_ride_refurbish_events = { nullptr,
```

Comment:
Not a fan of this indentation being dependant on the variable name

Reply:
Trailing commas at the end of the list avoid this. I've added these to all places I found, but skipped the windows folder.